### DISCUSSION TITLE
about late token (via email)

### DISCUSSION THREAD

**Discussion by nahyeonkaty** [seominjoon/kaist-ai605#37] [Q&A] [answered]
question:
1) If I want to use one token for assignment 2, can I just do late submission on KLMS or do I need to write elsewhere that I am using a token?
2) If i revise my code after late submission within token-extended day, do I need to re-submit it?

answer from professor:
1) You don't have to mention whether you use the credit or not in each assignment, because all late days will combined together at the end and deducted from your final score.
2) Please re-submit on KLMS with the same link. We download the notebooks right after the deadline, so we cannot know if it got updated unless you resubmit it again.

**Comment by seominjoon** [comment, marked as answer]
Thanks!
